File: Services/Dto/ServiceBudgetBench/SolutionPriceDto.cs
using System;

namespace Services.Dto.ServiceBudgetBench

{
    public class SolutionPriceDto
    {
        public int Id { get; set; }
        public DateTime DateService { get; set; }
        public string Technician { get; set; }
        public decimal PriceService { get; set; }
        public string ProblemByTechnician { get; set; }
        public string TechnicalSolution { get; set; }
        public bool Remote { get; set; }
        public bool Authorized { get; set; }
        public int ServiceBudgetId { get; set; }
        public ServiceBudgetDto ServiceBudget { get; set; }
    }
}

File: Views/src/components/services-provision/dtos/solution-price-dto.ts
import { ServiceBudgetDto } from "../budget/dto/service-budget-dto";

export class SolutionPriceDto {
  id: number;
  dateService: Date;
  technician: string;
  priceService: number; //only apear in budget
  problemByTechnician: string;
  technicalSolution: string;
  remote: boolean;
  // solved: boolean;
  approved: boolean;
  serviceBudgetId: number;
  serviceBudget: ServiceBudgetDto;
  // authorized: boolean; //only apear in budget
}
